### PR TITLE
Feature/cal 036 Boilers Redux. First look

### PR DESCRIPTION
**Pull request by alejandogodoy**

### CODE REVIEW

File: client/src/redux/actions/Boilers.actions.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  POST_BOILERS_REJECTED,
+  PUT_BOILERS_PENDING,
+  PUT_BOILERS_FULFILLED,
+  PUT_BOILERS_REJECTED,
+  DELETE_BOILERS_PENDING,
+  DELETE_BOILERS_FULFILLED,
+  DELETE_BOILERS_REJECTED,
+} from '../types/Boiler.types';
+
+const URL = 'http://localhost:5000/api/biolers';
```
Comment (tomascravero) [inline client/src/redux/actions/Boilers.actions.js]: URL must be 'http://localhost:5000/api/boilers'

File: client/src/components/pages/boilers/AddBoiler.jsx [hunk 2]
```diff
@@ -41,58 +34,61 @@ const AddBoiler = (props) => {
 
   return (
     <>
-      <button type="button" onClick={handleOpen}>
-        New Boilers
+      <button className="add-btn" type="button" onClick={handleOpen}>
```
Comment (tomascravero) [inline client/src/components/pages/boilers/AddBoiler.jsx:37]: function 'handleOpen' is not declared

File: client/src/components/pages/boilers/EditBoiler.jsx [hunk 3]
```diff
@@ -27,14 +26,17 @@ const EditBoiler = (props) => {
   };
 
   return (
-    <>
-      <button onClick={handleOpen}>Edit Boilers</button>
+    <div className="edit-container">
+      <button className="add-btn" onClick={handleOpen}>
```
Comment (tomascravero) [inline client/src/components/pages/boilers/EditBoiler.jsx]: function 'handleOpen' is not declared

File: client/src/components/pages/boilers/EditBoiler.jsx [hunk 4]
```diff
@@ -44,31 +46,31 @@ const EditBoiler = (props) => {
...
-            name="monthlyHorus"
+            name="Monthly Hours"
             onChange={handleChange}
           />
 
           <button type="submit" onClick={handleSubmit}>
             Edit Boiler
           </button>
-          <button type="submit" onClick={() => props.setEditing(false)}>
+          <button type="submit" onClick={handleClose}>
```
Comment (tomascravero) [inline client/src/components/pages/boilers/EditBoiler.jsx:69]: function 'handleClose' is not declared

File: client/src/components/pages/boilers/Boilers.jsx [hunk 5]
```diff
@@ -2,9 +2,13 @@ import React, { useState } from 'react';
 import boilers from './Boilers.json';
 import BoilersTable from './BoilersTable';
 import AddBoiler from './AddBoiler';
-import EditBoiler from './EditBoiler';
+import './Boilers.css';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 
-const Boilers = () => {
+import { postBoiler as postBoilerB } from '../../../redux/actions/Boilers.actions';
```
Comment (Ignacio-Sormani) [inline client/src/components/pages/boilers/Boilers.jsx]: PostBoilerB is not descriptive. If you would want it to have an alias name it should be something like PostBoilerAction

File: client/src/components/pages/boilers/BoilersTable.jsx [hunk 6]
```diff
@@ -1,41 +1,78 @@
...
+const mapDispatchToProps = (dispatch) =>
+  bindActionCreators(
+    {
+      getBoilers: getBoilersB,
+      deleteBoiler: deleteBoilerB,
+      putBoiler: putBoilerB,
+    },
+    dispatch
+  );
+const mapStateToProps = (state) => ({
```
Comment (Ignacio-Sormani) [inline client/src/components/pages/boilers/BoilersTable.jsx]: Add a white line between both maps

**Review by Alfondalix — APPROVED** [review]

**Review by tomascravero — APPROVED** [review]

**Pull request merged**